src/client/components/ResultMessageModalWindow/ResultTask.component.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './ResultMessageModalWindow.styles.css';
+import PropTypes from 'prop-types';
+
+export default function ResultTask({ correctAnswer }) {
+  if (correctAnswer > 3) {
+    return (
+      <div className="success_task">
+        <h1>Congratulations!</h1>
```

Comment:
You should not use h1 here, it is not a good practice to use h1 for a regular heading, use h3 instead. And In the actual design Incorrect answer and Congratulations is orange but i do not see orange color here.

Reply:
maby you are write, butI think here h1 or h3 doesn't make sense because I make control of it with css. 
@senner007 could you please help with ....
and the color was change it .. thanks for notice that ... :+1:

Reply:
Perfect, color is orange now 👍 And we should avoid h1 in the webpage other then the app title it is not good practice to use it other then that css has nothing to do with it. And h2 is reserved for the big headings like page heading in our case. You can google the h1, h2, h3 uses if not satisfied with my comments It is good to be curious 🙂